test/IRGen/class.sil

```diff
@@ -140,7 +140,7 @@ sil_vtable NonRequiredBase {
 }
 
 sil_vtable RequiredBase {
-  #NonRequiredBase.init!initializer.1: @init_RequiredBase
+  #NonRequiredBase.init!initializer.1: @init_RequiredBase [override]
```

Comment:
Isn't everything that has a different base type an override?

Reply:
Discussed offline